Aquality.Selenium.Template/Aquality.Selenium.Template/Utilities/CustomImageComparator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Aquality.Selenium.Core.Configurations;
+using Aquality.Selenium.Core.Visualization;
+using System.Drawing;
+
+namespace Aquality.Selenium.Template.Utilities
+{
+    public static class CustomImageComparator
+    {
+        private static IVisualizationConfiguration visualizationConfiguration;
+
+        public static void Init(float defaultThreshold = 0.1F, int comparisonWidth = 20, int comparisonHeight = 20)
```

Comment:
why are we hardcoding the values, is it a good example for template?

Reply:
The code was updated